fix compatibility with JasPer 2.x
    enc_jpeg2000.c:141:10: error: jas_image_t has no member named inmem_
see also https://github.com/wrf-model/WPS/pull/50
From e23dbcdb130a2a6306f88e08af3b4c853269a86d Mon Sep 17 00:00:00 2001
From: Dave Gill <gill@ucar.edu>
Date: Wed, 6 Dec 2017 21:07:22 -0700
Subject: [PATCH] image.inmem_=1 causes a compiler error, not fatal to ungrib

---
 ungrib/src/ngl/g2/enc_jpeg2000.c | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/ungrib/src/ngl/g2/enc_jpeg2000.c b/ungrib/src/ngl/g2/enc_jpeg2000.c
index 7b8528120..2be6c062b 100755
--- a/ungrib/src/ngl/g2/enc_jpeg2000.c
+++ b/ungrib/src/ngl/g2/enc_jpeg2000.c
@@ -138,7 +138,10 @@ int enc_jpeg2000(unsigned char *cin,g2int *pwidth,g2int *pheight,g2int *pnbits,
     image.clrspc_=JAS_CLRSPC_SGRAY;         /* grayscale Image */
     image.cmprof_=0; 
 #endif
-    image.inmem_=1;
+ /* 
+  * Does not seem to be needed, and throws a compiler error
+  * image.inmem_=1;
+  */
 
     cmpt.tlx_=0;
     cmpt.tly_=0;
